src/core/ext/transport/cronet/plugin_registry/grpc_cronet_plugin_registry.cc

```diff
@@ -30,12 +30,9 @@ void grpc_client_channel_init(void);
 void grpc_client_channel_shutdown(void);
 
 void grpc_register_built_in_plugins(void) {
-  grpc_register_plugin(grpc_http_filters_init,
-                       grpc_http_filters_shutdown);
-  grpc_register_plugin(grpc_chttp2_plugin_init,
-                       grpc_chttp2_plugin_shutdown);
+  grpc_register_plugin(grpc_http_filters_init, grpc_http_filters_shutdown);
```

Comment:
Why😄? Does the formatter enforce difference column width in the two directories?

Reply:
This is the result of clang-formatter. Previous source was generated from the build tool and it doesn't go though the formatter.